Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -402,9 +477,7 @@ public NodejsProjectNode(NodejsProjectPackage package)
                 var ignoredPaths = GetProjectProperty(NodejsConstants.AnalysisIgnoredDirectories);
 
                 if (!string.IsNullOrWhiteSpace(ignoredPaths)) {
-                    _analysisIgnoredDirs = ignoredPaths.Split(';').Select(x => '\\' + x + '\\').ToArray();
-                } else {
-                    _analysisIgnoredDirs = new string[0];
+                    _analysisIgnoredDirs.Append(ignoredPaths.Split(';').Select(x => '\\' + x + '\\').ToArray());

```

Comment:
Is _analysisIgnoredDirs just a list? How often is this function called? Is there a risk here of repeatedly appending to the list over time? (If it's a set, this seems safe).